File: 3498-reverse-degree-of-a-string/3498-reverse-degree-of-a-string.java
class Solution {
    public int reverseDegree(String s) {
        int sum = 0;

        for (int i=0; i<s.length(); i++) {
            int ind = 27 - (s.charAt(i) - 'a' + 1);

            sum += ind * (i + 1);
        }

        return sum;
    }
}

File: 3498-reverse-degree-of-a-string/3498-reverse-degree-of-a-string.cpp
class Solution {
public:
    int reverseInt (char ch) {
        int c = ch - 'a';

        return 26 - c;
    }
    int reverseDegree(string s) {
        int ans = 0;

        for (int i=0; i<s.size(); i++) {
            int ind = i+1;

            ans += ind *  reverseInt (s[i]);
        }

        return ans;
    }
};